crates/transaction-emitter-lib/src/emitter/account_minter.rs

```diff
@@ -150,6 +150,15 @@ impl<'t> AccountMinter<'t> {
             }
         }
 
+        let new_source_account = if !req.delay_after_minting.is_zero() {
```

Comment:
I am not sure if we should couple `delay_after_minting` with new source account feature. How about having a separate flag for it? Or better just make it default? Is there any downside?

Reply:
Only reason to create another intermediary account (to create seed accounts from), instead of using root/provided source account directly - is if there are multiple txn emitter instances running simultaneously. 

delay_after_minting is also pretty much needed only for coordinating multiple txn emitter instances.


So I think having a single flag is pretty reasonable, and simplifies configuration.

documentation for the flag explain this:

```
    // In cases you want to run txn emitter from multiple machines,
    // and want to make sure that initialization succeeds
    // (account minting and txn-specific initialization), before the
    // loadtest puts significant load, you can add a delay here.
    //
    // This also enables few other changes needed to run txn emitter
    // from multiple machines simultaneously:
    // - retrying minting phase before this delay expires
    // - having a single transaction on root/source account
    //   (to reduce contention and issues with sequence numbers across multiple txn emitters).
    //   basically creating a new source account (to then create seed accounts from).
```

Reply:
In that case, may change the config name `delay_after_minting` to something like `coordination_delay_for_multiple_run` ?